discover/deployment/articles/06-upgrading-liferay/02-preparing-for-upgrade.markdown

```diff
@@ -140,28 +140,31 @@ It's time to migrate and update your document store configuration to
     to one of the other store options before upgrading from your current Liferay
     version. 
 
--   Since @product@ 7.0, document store specific configuration (e.g., 
+1. If you're using Advanced File System Store or Simple File System Store, copy the document ling Advanced File System Store or Simple File System Store, copy the document library's root folder to your new installation or make sure the the newibrary's root folder to your new installation or make sure the the new installation can access it.
```

Comment:
Same thing as #4933. Looks like copy/pasta error.

Reply:
OK. It's fixed now and ready for review. Thanks!